mantle/kola/harness.go

```diff
@@ -761,6 +761,16 @@ func registerTestDir(dir, testprefix string, children []os.FileInfo) error {
 			if err := registerTestDir(subdir, subprefix, subchildren); err != nil {
 				return err
 			}
+		} else if isreg && (c.Mode().Perm()&0001) == 0 {
+			file, err := os.Open(filepath.Join(dir, c.Name()))
+			if err != nil {
+				return errors.Wrapf(err, "opening %s", c.Name())
+			}
+			scanner := bufio.NewScanner(file)
+			scanner.Scan()
+			if strings.HasPrefix(scanner.Text(), "#!") {
+				plog.Warningf("WARN: Found non-executable file with shebang: %s \n", c.Name())
```

Comment:
I think we can drop the `WARN:` prefix since that's already signaled by using `Warningf()`. :)  The formatter we use right now doesn't print log levels though, but we could tweak that.

Also looks like extra space and newline at the end there.

```suggestion
				plog.Warningf("Found non-executable file with shebang: %s", c.Name())
```